File: backend/app/src/models/population/Baker.py
from .Worker import Worker


class Baker(Worker):
    def __init__(self, name, employed = True, working = False, production: int = 500):
        super().__init__(name=name, employed=employed, field_of_work="baker", working=working, production=production)

    # def fetch_population_state(self) -> dict:
    #     population_state: dict = super().fetch_population_state()
    #     population_state.update({

    #     })
        
    #     return population_state
    

File: frontend/static/script/models/population/Baker.js
"use strict";
import { Worker } from "./Worker.js";



export class Baker extends Worker {
    constructor(population_id, name, profession, alive, employed, field_of_work, working, production) {
        super(population_id, name, profession, alive, employed, field_of_work, working, production);
        
    }
}
